doc/rados/operations/health-checks.rst

```diff
@@ -1640,6 +1640,19 @@ We encourage you to fix this by making the weights even on both dividing buckets
 This can be done by making sure the combined weight of the OSDs on each dividing
 bucket are the same.
 
+NONEXISTENT_MON_CRUSH_LOC_STRETCH_MODE
+______________________________________
+
+The CRUSH location specified for the monitor must belong to one of the dividing
+buckets when stretch mode is enabled. With the ``tiebreaker`` monitor being the
+only exception.
```

Comment:
Comma not period.  The latter part here is not a sentence.

Reply:
@anthonyeleven Ah okay, I guess I'll file a new PR for this.

Reply:
or I can do so, let's get in it while we can.

Reply:
@anthonyeleven okay I think you'll be faster, thank you!